openwisp_notifications/handlers.py

```
@@ -163,9 +165,18 @@ def notify_handler(**kwargs):
         if kwargs:
             notification.data = kwargs
-        notification.save()
-        notification_list.append(notification)
-
+        notifications_to_create.append(notification)
+    notification_list = Notification.objects.bulk_create(notifications_to_create)
+    for notification in notification_list:
+        send_email_notification(Notification, notification, created=True)
+    for recipient in recipients_list:
+        Notification.invalidate_unread_cache(recipient)
+    notification_map = dict(zip(recipients_list, notification_list))
```

Ruff (0.14.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)